metadata.json

```diff
@@ -28,6 +28,14 @@
       "name": "puppetlabs/java_ks",
       "version_requirement": ">= 1.0.0 <4.0.0"
     },
+    {
+      "name": "puppetlabs/augeas_core",
+      "version_requirement": ">= 1.0.0 <4.0.0"
+    },
+    {
+      "name": "puppetlabs/yumrepo_core",
+      "version_requirement": ">= 1.0.0 <4.0.0"
```

Comment:
This should likely be `>= 1.0.0 <2.0.0`.  I'm not certain this is even needed though as it appears like fresh install of `puppet-agent` pulls in a copy of this module.

Reply:
> Added a few more things to change, acceptance tests should now be possible to pass if things actually work, the bug with Puppet has been fixed and new released pushed to repos.

What you requested have now been done, I believe.